### DISCUSSION TITLE
After recent Google Chrome update wavesurfer.js elements disappear from tab when you left it

### DISCUSSION THREAD

**Discussion by ivantalaychuk** [katspaugh/wavesurfer.js#3710] [Q&A]
Hi!

After recent chrome update I noticed that in all services where wavesurfer.js is present, it disappears when you switch to another tab and then go back to same tab with waveform.

Who also found this issue? How did you overcome it?

Thanks!

**Comment by katspaugh** [comment]
Can you reproduce this on https://wavesurfer.xyz/? I have Chrome version 125 and for me nothing disappears.

**Comment by vikhyatsingh123** [comment]
@ivantalaychuk Yes for me too, It's happening. And when you zoom in or zoom out, only then waveforms comes back

**Comment by entonbiba** [comment]
@ivantalaychuk could be a bug in some of the older subversions of chrome, just tested with the latest beta and stable versions as of today and doesn't seem to have the issue.
![image](https://github.com/katspaugh/wavesurfer.js/assets/5193884/6d1edb93-0387-4b4a-a719-3369091a825e)
![image](https://github.com/katspaugh/wavesurfer.js/assets/5193884/f52e3154-ef0d-4cb8-9765-f6b46593efff)
